# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerPrivate.h
# Commit: ffad31a6d765
# Full Hash: ffad31a6d765162da71b8c2ab272447f58ffdc48
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-03-18 21:32:34
# Regressor Bug: 1899503
# File Overlap Count: 1
# Description:
#   Bug 1899503 - P4 Update SharedWorker windowID for remote worker debugger. r=asuth
#   
#   Depends on D230260
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231778
# ==============================================================================

diff -r b0cfee4dccb7 -r ffad31a6d765 dom/workers/WorkerPrivate.h
--- a/dom/workers/WorkerPrivate.h	Tue Mar 18 12:10:54 2025 +0000
+++ b/dom/workers/WorkerPrivate.h	Tue Mar 18 12:10:54 2025 +0000
@@ -1078,6 +1078,8 @@
 
   bool IsQueued() const;
 
+  void UpdateWindowIDToDebugger(const uint64_t& aWindowID, const bool& aIsAdd);
+
   already_AddRefed<WorkerRunnable> MaybeWrapAsWorkerRunnable(
       already_AddRefed<nsIRunnable> aRunnable);
 